src/components/app/app.tsx

```diff
@@ -1,12 +1,39 @@
+import {BrowserRouter, Route, Routes} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../const.ts';
+
 import MainPage from '../../pages/main/main.tsx';
+import LoginPage from '../../pages/login/login.tsx';
```

Comment:
Давай посмотрим как в vite настроить алиас для путей чтобы было 
```suggestion
import LoginPage from 'pages/login/login.tsx';
```